src/main/java/jenkins/metrics/impl/JenkinsMetricProviderImpl.java

```diff
@@ -1018,7 +1018,7 @@ private void checkEnterQueue(Queue.Item i) {
                         null,
                         null);
                 executorService.submit(() -> {
-                    QueueItemMetricsListener.notifyStarted(m);
+                    QueueItemMetricsListener.notifyQueued(m);
```

Comment:
This has the potential to be a breaking change, but from [a quick GitHub search](user:jenkinsci QueueItemMetricsListener) it looks like there aren't any open source plugins using `QueueItemMetricsListener` in the first place, so the risk seems limited.